docker-compose.yml

```diff
@@ -166,7 +166,7 @@ services:
 
   preprints:
     image: quay.io/centerforopenscience/ember-preprints:develop
-    command: /bin/bash -c "npm install && ./node_modules/bower/bin/bower install --allow-root --config.intedractive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4200"
+    command: /bin/bash -c "yarn --pure-lockfile --no-progress --no-emoji && ./node_modules/bower/bin/bower install --allow-root --config.interactive=false && ./node_modules/ember-cli/bin/ember serve --host 0.0.0.0 --port 4200"
```

Comment:
Interesting! And yes, I didn't look at the line above closely enough.

So we're not quite at the point of using Yarn for the OSF yet- just making sure that preprints obeys the settings we used elsewhere.

It's interesting that we need this command in addition to the Dockerfile. Still getting a handle on the interdependencies in different repos. Thanks.